Release 4.3.0 changelog edits (#444)
NordicSemiconductor/pc-nrfconnect-ppk: Changelog.md

## 4.0.0 - 2024-03-13

### Added

> -   Reintroduced triggers functionality.

Reviewer: Will the user know what the triggers functionality is?

Author: yes

> -   The configuration in the advance menu to stop saving to disk when disk is

Reviewer: ```suggestion
-   The configuration in the advanced settings menu to stop saving to hard disk when the HDD space is
```

> -   The configuration in the advanced settings menu to stop saving to hard disk

Reviewer 2: ```suggestion
-   Configuration in the advanced settings menu to stop saving to hard disk
```

>     running low on disk space.

Reviewer: ```suggestion
    is running low.
```

### Changed

-   Changed export format to `.ppk2`. `.ppk` has been deprecated and importing
    this format will autogenerate a `.ppk2` file next to the `.ppk` file.
-   Removed limitations for recording duration by using the disk to store the
    collected data instead of using RAM.
-   Performance improvements.

> -   Default path for the session is in the temp folder. This can be changed from

Reviewer: ```suggestion
-   The temporary folder is now the default path for the session. This can be changed from
```

Reviewer 2: he temporary folder is the default path for the session. This can be changed from

> -   Performance improvements.

Reviewer 2: - The advanced Settings are always visable on the side panel.

>     the advanced menu.

Reviewer: ```suggestion
    the advanced settings menu.
```

### Fixed

> -   Chart drifting in live mode.

Reviewer 2: My concern here is that we sole one instance of this from happening ... I know there are other cases that we were not able to solve unless we do a FW update

Author: Fixed:
* Some cases when chart was drifting in the live mode.
--
Maybe like this?

## 4.0.0 - 2024-03-13

### Added

-   Reintroduced triggers functionality. Triggers can be automatically saved to
    the file. If too many triggers occur the application may turn off auto
    export to safeguard itself.
-   The configuration in the advance menu to stop saving to disk when disk is
    running low on disk space.

> ### Changed

Reviewer 2: Duplicate
